tree_test.go

```diff
@@ -1152,60 +1153,17 @@ func TestBatchMigratedKeyValues(t *testing.T) {
 						t.Fatalf("failed to insert key: %v", err)
 					}
 				}
-				unbatchedRoot := tree.Commit().Bytes()
+				tree.Commit()
 				if _, err := tree.(*InternalNode).BatchSerialize(); err != nil {
 					t.Fatalf("failed to serialize unbatched tree: %v", err)
 				}
 				unbatchedDuration += time.Since(now)
-
-				// ***Insert the key pairs with optimized strategy & methods***
-				rand = mRand.New(mRand.NewSource(42)) //skipcq: GSC-G404
-				tree = genRandomTree(rand, treeInitialKeyValCount)
-				randomKeyValues = genRandomKeyValues(rand, migrationKeyValueCount)
-
-				now = time.Now()
-				// Create LeafNodes in batch mode.
-				nodeValues := make([]BatchNewLeafNodeData, 0, len(randomKeyValues))
-				curr := BatchNewLeafNodeData{
-					Stem:   randomKeyValues[0].key[:StemSize],
-					Values: map[byte][]byte{randomKeyValues[0].key[StemSize]: randomKeyValues[0].value},
-				}
-				for _, kv := range randomKeyValues[1:] {
-					if bytes.Equal(curr.Stem, kv.key[:StemSize]) {
-						curr.Values[kv.key[StemSize]] = kv.value
-						continue
-					}
-					nodeValues = append(nodeValues, curr)
-					curr = BatchNewLeafNodeData{
-						Stem:   kv.key[:StemSize],
-						Values: map[byte][]byte{kv.key[StemSize]: kv.value},
-					}
-				}
-				// Append last remaining node.
-				nodeValues = append(nodeValues, curr)
-
-				// Create all leaves in batch mode so we can optimize cryptography operations.
-				newLeaves := BatchNewLeafNode(nodeValues)
-				if err := tree.(*InternalNode).InsertMigratedLeaves(newLeaves, nil); err != nil {
-					t.Fatalf("failed to insert key: %v", err)
-				}
-
-				batchedRoot := tree.Commit().Bytes()
-				if _, err := tree.(*InternalNode).BatchSerialize(); err != nil {
-					t.Fatalf("failed to serialize batched tree: %v", err)
-				}
-				batchedDuration += time.Since(now)
-
-				if unbatchedRoot != batchedRoot {
-					t.Fatalf("expected %x, got %x", unbatchedRoot, batchedRoot)
-				}
```

Comment:
All this is gone since now we don't have a base API vs batch API anymore; the fast version is the base API.
We can evaluate later if it makes sense to keep this test/benchmark; but for now can give useful information as shown in the PR description.